webpack/scenes/ContentViews/Details/Filters/CVFilterDetailType.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CVPackageGroupFilterContent from './CVPackageGroupFilterContent';
+import CVRpmFilterContent from './CVRpmFilterContent';
+
+const CVFilterDetailType = ({
+  cvId, filterId, inclusion, type,
+}) => {
+  if (type === 'package_group') {
```

Comment:
Are there any more types besides `package_group` and `rpm`?  Maybe this would be better as a case statement..

Reply:
There are but these are the only 2 implemented in the new UI.
There are errata, module stream and docker filters apart from rpm and package group filters..

Reply:
yeah I'd say a `switch` statement is appropriate then!